infrastructure/tooling/src/smoketest/checks/ping.js

```diff
@@ -34,7 +34,7 @@ tasks.push({
     `ping-web-server`,
   ],
   run: async (requirements, utils) => {
-    const serverHealth = `${process.env.TASKCLUSTER_ROOT_URL}/.well-known/apollo/server-health`;
+    const serverHealth = `${process.env.TASKCLUSTER_ROOT_URL}/api/web-server/v1/ping`;
```

Comment:
new  apollo server doesn't come with healthcheck endpoints, so we revert back to the app's ping endpoint

Otherwise web-server-web container wouldn't start in kubernetes